gtsam/hybrid/HybridBayesNet.h

```diff
@@ -36,6 +38,35 @@ class GTSAM_EXPORT HybridBayesNet : public BayesNet<HybridConditional> {
 
   /** Construct empty bayes net */
   HybridBayesNet() = default;
+
+  /// Add HybridConditional to Bayes Net
+  using Base::add;
+
+  /// Add a discrete conditional to the Bayes Net.
+  void add(const DiscreteKey &key, const std::string &table) {
```

Comment:
I am not in favor of more helpers, do we have similar stuff in `DiscreteBayesNet`?